llvm/test/DebugInfo/NVPTX/debug-bool-var.ll

```diff
@@ -12,15 +12,8 @@ entry:
   ;     foo(xyz);
   ;   }
   ;
-  ; Verify that debug info exists for "xyz" variable
-  ;
-  ; CHECK:       DW_TAG_variable
-  ; CHECK:      .b8 120     // DW_AT_name
-  ; CHECK-NEXT: .b8 121
-  ; CHECK-NEXT: .b8 122
-  ; CHECK-NEXT: .b8 0
-  ; CHECK-NEXT: .b8 1       // DW_AT_decl_file
-  ; CHECK-NEXT: .b8 6       // DW_AT_decl_line
+  ; CHECK-LABEL: Machine code for function test1
+  ; CHECK: DBG_VALUE %{{[0-9]+}}:int32regs, $noreg, !"xyz", !DIExpression(), debug-location !13; test.cu:2 line no:6
```

Comment:
One final nit - please can you avoid hard coding the `debug-location` hard coded metadata number (you could use `[[#]]`  or omit the rest of the line, etc). Same below. This reduces spurious failures if the metadata numbers shift around as a result of a different change.

Reply:
Reasonable concern, absolutelly agree. Fixed.